templates/store/snap-details.html

```diff
@@ -272,6 +291,40 @@ <h4>Users by distribution (log)</h4>
     {% include "store/snap-details/_report_snap_modal.html" %}
   {% endif %}
 
+  {% if has_publisher_page %}
```

Comment:
Wasn't that supposed to be below the map? So that "Report snap" is at the end of the page?

Reply:
Looking at @kwm14 design review, it seems like featured snaps should be in the end of the page?

Reply:
Yes, sorry, I missunderstood his comment.